app/controllers/shopify_app/sessions_controller.rb

```diff
@@ -46,6 +46,7 @@ def login_shop
       sess = ShopifyAPI::Session.new(shop_name, token)
       session[:shopify] = ShopifyApp::SessionRepository.store(sess)
       session[:shopify_domain] = shop_name
+      session[:shopify_user] = associated_user if associated_user.present?
```

Comment:
That causing `NoMethodError: undefined method `session_options' for nil:NilClass` Can we open an issue to tackle this later?